README.md

```diff
@@ -38,7 +38,8 @@ stampy from the test discord server and see the messages in your terminal.
 
 # How to Contribute
 
-Checkout the [Trello](https://trello.com/b/LBmYgkes/stampy). It's a great place to start. Make sure to attach your name to the card you're going to work on and then move it into the progress lane. Currently, there are still many setup and organizational tasks. In the future, we would mostly want Stampy contributions to be in the form of adding new modules.
+Check out the currently open github issues and pull requests, if you see something open you can help out with add a comment. Most coordinations is done through live voice calls in the discord.
+There's a [Trello](https://trello.com/b/LBmYgkes/stampy) you can request access to, but it's not very updated.
```

Comment:
feel free to remove any mention of Trello, it's not just `not very updated`, it's not used at all, thanks!

(@plexish  still uses it for other stuff, but not any more for this repo, we just forgot to update README)

Reply:
oh, it's merged already, nevermind, I will update README to remove trello myself

Reply:
ye I was a bit surprised it was merged near-instantly